app/main.py

```diff
@@ -2,0 +1,40 @@
+from __future__ import annotations
+
+
+class Animal:
+    alive: list = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def is_alive(self) -> bool:
+        return self.health > 0
+
+    @staticmethod
+    def remove_dead(animal: Herbivore) -> None:
+        if not animal.is_alive():
+            Animal.alive.remove(animal)
+
+    def __repr__(self) -> str:
+        return (
+            f"{{Name: {self.name}, "
+            f"Health: {self.health}, "
+            f"Hidden: {self.hidden}}}"
```

Comment:
Code Style: Use interpolation instead of concatenation. Instead of concatenating strings with `+`, use f-string formatting. It's more efficient and easier to read.